test/tests/meshgenerators/poly2tri_mesh_generator/poly2tri_from_1d_sbdids.i

```diff
@@ -0,0 +1,38 @@
+[Mesh]
```

Comment:
We need to get you an editor that has HIT formatting! [./] is old syntax

Reply:
Sure looks nicer after getting rid of those.

But this is another "need to go through and change it everywhere".  I've read Stroustrop cover-to-cover before, but I'm afraid for any less common language I'm going to cheat by inferring syntax from existing examples rather than reference docs ... and that just doesn't work here, because `find -name '*.i' | xargs grep -l '\[./' | wc -l` is still a couple thousand hits.

Reply:
The HIT formatter is almost done but isn't 100% yet. It's kinda broken on string vectors. Once it's ready, we'll use a service account to bulk format and then require formatting. Until then, we just ask that new inputs be formatted. Not a requirement

Reply:
> Not a requirement

Oh, it's a decent request.  Already done in my branch.  Just wondering if I should set up another PR with the results of hitting those other thousand hits with a perl script.  If we've got a better fix in the works then it can wait for that.